indico/web/client/js/custom_elements/ind_datepicker.js

```diff
@@ -0,0 +1,631 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import './ind_datepicker.scss';
+
+const TODAY = (function() {
```

Comment:
I know it's a somewhat rare edge case, but this will be incorrect when the day changes between loading the page and opening the date picker ;)

Reply:
True.